src/data_provider/src/osinfo/sysOsParsers.cpp

```diff
@@ -137,15 +124,16 @@ static bool findVersionInStream(std::istream& in,
 bool UnixOsParser::parseFile(std::istream& in, nlohmann::json& info)
 {
     constexpr auto SEPARATOR{'='};
-    static const std::map<std::string, std::string> KEY_MAP
+    static const std::vector<std::pair<std::string, std::string>> KEY_MAPPING
```

Comment:
The changes made to this function, while elementary, exceed the original scope of the issue